frida/core.py

```diff
@@ -179,6 +182,29 @@ def __dir__(self) -> List[str]:
         return self._script.list_exports()
 
 
+class ScriptExportsAsync:
+    """
+    Proxy object that expose all the RPC exports of a script as attributes on this class
+
+    A method named exampleMethod in a script will be called with instance.example_method on this object
+    """
+
+    def __init__(self, script: "Script") -> None:
+        self._script = script
+
+    def __getattr__(self, name: str) -> Callable[..., Awaitable[Any]]:
+        script = self._script
+        js_name = _to_camel_case(name)
+
+        async def method(*args: Any, **kwargs: Any) -> Any:
+            return await script._rpc_request_async("call", js_name, args, **kwargs)
+
+        return method
+
+    def __dir__(self) -> List[str]:
+        return self._script.list_exports()
```

Comment:
This should use the new `list_exports_sync` method to avoid the deprecation warning.
```suggestion
        return self._script.list_exports_sync()
```